pkgs/by-name/na/navidrome/package.nix

```diff
@@ -55,14 +56,17 @@ buildGo124Module rec {
   ];
 
   ldflags = [
-    "-X github.com/navidrome/navidrome/consts.gitSha=${src.rev}"
-    "-X github.com/navidrome/navidrome/consts.gitTag=v${version}"
+    "-X github.com/navidrome/navidrome/consts.gitSha=${finalAttrs.src.rev}"
+    "-X github.com/navidrome/navidrome/consts.gitTag=v${finalAttrs.version}"
   ];
 
   CGO_CFLAGS = lib.optionals stdenv.cc.isGNU [ "-Wno-return-local-addr" ];
 
   postPatch = ''
     patchShebangs ui/bin/update-workbox.sh
+
+    # Allow older go versions
+    sed -i go.mod -e 's/^go .*/go ${finalAttrs.passthru.go.version}/'
```

Comment:
I get your sentiment. People pinning patch versions in go is stupid but we should keep the minor version to avoid any weird compile errors no one can easily decipher.